lib/cpp/src/thrift/transport/TSSLSocket.cpp

```diff
@@ -140,13 +145,11 @@ SSLContext::SSLContext(const SSLProtocol& protocol) {
   if(protocol == SSLTLS)
   {
     ctx_ = SSL_CTX_new(SSLv23_method());
-  }
-  else if(protocol == SSLv3)
-  {
+#ifndef OPENSSL_NO_SSL3
```

Comment:
There is no such macro in https://github.com/apache/thrift/pull/1102/files change.

Reply:
Oh wrong commit. Hold on...